launch_safety_tree: false
launch_shutdown_tree: false
battery_percent_window_len: 6
lights:
  critical_battery_anim_period: 15.0
  critical_battery_threshold_percent: 0.1
  battery_state_anim_period: 120.0
  low_battery_anim_period: 30.0
  low_battery_threshold_percent: 0.4
  update_charging_anim_step: 0.1
plugin_libs:
  - tick_after_timeout_bt_node
ros_plugin_libs:
  - call_set_led_animation_service_bt_node
